content/applications/services/helpdesk/overview/receiving_tickets.rst

```diff
@@ -206,72 +147,129 @@ of when a :guilabel:`URL Regex` action is triggered (e.g., a page visit).
+   conditional *Only If* logic that applies when certain pre-filled answers are chosen. To create
+   more steps in the script, click :guilabel:`Add a line` and fill out the script steps form
+   according to the desired logic.
+
+Add the URLs for the pages where the channel should appear to in the :guilabel:`URL Regex` field. If
+this channel should only be available to users in specific countries, add them to the
 :guilabel:`Country` field. If this field is left blank, the channel will be available to all site
 visitors.
 
 .. image:: receiving_tickets/receiving-tickets-channel-rules.png
    :align: center
-   :alt: View of the kanban cards for the available Live Chat channels.
+   :alt: View of the Kanban cards for the available Live Chat channels.
 
 Use the live chat widget
 ************************
 
-The :guilabel:`Widget` tab on the live chat channel form offers an embeddable website widget, or a
-shortcode for instant customer/supplier access to a live chat window.
+The :guilabel:`Widget` tab on the live chat channel form offers a website widget that can be added
+to third party platforms. Additionally, a short code is available, that can provide instant access
+to a live chat window.
 
 The live chat :guilabel:`Widget` can be applied to websites created through Odoo by navigating to
-the :menuselection:`Website --> Configuration --> Settings`. Then scroll to the :guilabel:`Live
-Chat` section, and select the channel to add to the site. Click :guilabel:`Save` to apply.
+the :menuselection:`Website app --> Configuration --> Settings --> Email & Marketing`. Then scroll
+to the :guilabel:`Live Chat` section, and select the channel to add to the site. Click
+:guilabel:`Save` to apply.
 
-To add the widget to a website created on a third-party platform, click :guilabel:`COPY` and paste
+To add the widget to a website created on a third-party platform, click :guilabel:`Copy` and paste
 the code into the `<head>` tag on the site.
 
-Likewise, to send a live chat session to a customer or supplier, click the second :guilabel:`COPY`
+Likewise, to send a live chat session to a customer or supplier, click the second :guilabel:`Copy`
 button which contains a link to join directly.
 
 Create a support ticket from a live chat session
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Once live chat is enabled, operators will be able to communicate with site visitors in real time.
+Once live chat is enabled, operators are able to communicate with site visitors in real time.
 
 During the conversation, an operator can use the shortcut :doc:`command
-</applications/websites/livechat/responses>` `/helpdesk` to create a ticket without leaving
-the chat window. The transcript from the conversation will be added to the new ticket, under the
+</applications/websites/livechat/responses>` `/ticket` to create a ticket without leaving the chat
+window. The transcript from the conversation is added to the new ticket, under the
 :guilabel:`Description` tab.
```

Comment:
Maybe also add WA tip in here too! I think it's important to add. 

.. tip::
The WhatsApp integration (app) also has a create a ticket feature similar to Livechat. Simply type in /ticket and a Helpdesk ticket will be created instantly.